prettyconf/loaders.py

```diff
@@ -336,6 +337,7 @@ def _fetch_parameters(self):
             aws_access_key_id=self.aws_access_key_id,
             aws_secret_access_key=self.aws_secret_access_key,
             region_name=self.region_name,
+            endpoint_url=self.endpoint_url,
```

Comment:
Is it ok to use `None` as an `endpoint_url` argument here?

Reply:
Yes, it is the default value:
https://boto3.amazonaws.com/v1/documentation/api/latest/_modules/boto3/session.html#Session.client